downloaders/dumps_downloader.py

```diff
@@ -0,0 +1,76 @@
+import urllib.request
```

Comment:
Please follow pep8 guide for [import order](https://www.python.org/dev/peps/pep-0008/#imports)